src/ShortcutsManager.cpp

```diff
@@ -19,6 +20,24 @@
 
 #include "ShortcutsManager.h"
 
+ShortcutsManager::~ShortcutsManager()
```

Comment:
Just a newbie question - so even if this class object is living for entire life of application it still needs destructor? Or is just good practice?

Reply:
in this particular instance I suspect it is just *good practice* - always clean up after yourself...

Reply:
BTW, why is the single instance of this class that is used created dynamically with `new` in the `mudlet` constructor? It is simpler to have a instance as a member of that class:
I.e. in the `mudlet.h` header you could change:
```diff
-    QPointer<ShortcutsManager> mShortcutsManager;
+    ShortcutsManager mShortcutsManager
```
Then in the constructor you do not need to create it (or explicitly destroy it - which isn't happening either at the moment):
```cpp
-    mShortcutsManager = new ShortcutsManager();
-    mShortcutsManager->registerShortcut(qsl("Script editor"), tr("Script editor"), &triggersKeySequence);
+    mShortcutsManager.registerShortcut(qsl("Script editor"), tr("Script editor"), &triggersKeySequence);
-    mShortcutsManager->registerShortcut(qsl("Show Map"), tr("Show Map"), &showMapKeySequence);
+    mShortcutsManager.registerShortcut(qsl("Show Map"), tr("Show Map"), &showMapKeySequence);
// etc...
```

As it is, the *current* name `mShortcutsManager` is a little confusing - it is actually a pointer (admittedly a *smart* one) and ***I*** (at least) like to have a `p` prefix for those as it helps to remember that outside of an IDE (like Qt Creator). If it had been my creation I would have called it `mpShortcutsManager` - of course if you follow the step I am hinting about above then the member would be the item concerned and NOT a pointer to it - so *then* it would be consistent to not have that `p` prefix. :grinning: